src/style.css

```diff
@@ -1,3 +1,69 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: "Helvetica", sans-serif;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: auto;
+  box-sizing: border-box;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav__link {
+  display: flex;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  padding-top: 23px;
+  padding-bottom: 22px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: normal;
+  text-align: center;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:active {
+  color: #00acdc;
+  border-bottom: #00acdc solid 4px;
+  border-radius: 1px;
+}
+
+.is-active:hover {
+  color: #00acdc;
+  border-bottom: #00acdc solid 4px;
+  padding-bottom: 19px;
+
+}
```

Comment:
Blue line under 'Apple' should be visible all the time. Furthermore it should be done with pseudo element like `::after`